os-certmonger.te

```diff
@@ -0,0 +1,14 @@
+policy_module(os-certmonger,0.1)
+
+gen_require(`
+  type certmonger_t;
+  type puppet_etc_t;
+  type container_runtime_exec_t;
+  type container_var_run_t;
+  type container_var_lib_t;
+  type container_runtime_tmpfs_t;
+  class dir {getattr search};
+')
```

Comment:
We should be able to clean up the gen_require as well, we don't need the container types for this bug now. Thank you!